doc/modules/model_evaluation.rst

```diff
@@ -117,7 +117,7 @@ Usage examples:
 
 .. note::
 
-    The values listed by the ValueError exception correspond to the functions measuring
+    The values listed by the ``ValueError`` exception correspond to the functions measuring
```

Comment:
Just  FYI we changed the meaning of single backquotes so that they have the same effect of double backquotes. I tend to use them because it's easier to read the plain text, but we don't have specific guidelines for this so do as you prefer

Reply:
Do single and double backquotes have different semantics?

Reply:
They are exactly the same now.